src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -1,10 +1,16 @@
 package frc.robot.subsystems;
 
 import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
 
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.wpilibj.simulation.DifferentialDrivetrainSim;
```

Comment:
You shouldn't need Velocity. Also remove any other unused imports if there are any.